system/classes/KO7/Controller/REST.php

```diff
@@ -0,0 +1,134 @@
+<?php
+/**
+ * Abstract Controller class for REST controller mapping.
+ * Supports GET, PUT, POST, and DELETE.
+ *
+ * @package        KO7\REST
+ *
+ * @copyright  (c) since 2016 Koseven Team
+ * @license        https://koseven.ga/LICENSE
+ */
+abstract class KO7_Controller_REST extends Controller {
+
+    /**
+     * Default Output (only if allowed by configuration)
+     *
+     * @var string
+     */
+    public static $default_output = 'json';
+
+    /**
+     * REST types
+     *
+     * @var array
+     */
+    protected static $_action_map = [
```

Comment:
why is static property?

Reply:
Probably because of a plugin from my IDE which converts "heavy" initialized variables to static ones..... after some research i have not really found a reason for this and asked (because i am curios) the developer why....maybe it is considered some kind of best practice...but anyway i am not really a fan of static stuff, i agree and moved it to non static